src/common/gui/SurgeBitmaps.cpp

```diff
@@ -72,7 +80,11 @@ void SurgeBitmaps::addEntry(int id)
 {
    assert(bitmap_registry.find(id) == bitmap_registry.end());
 
+#ifdef USE_SCALABLE_BITMAPS
+   VSTGUI::CBitmap *bitmap = new CScalableBitmap(CResourceDescription(id));
+#else
    VSTGUI::CBitmap* bitmap = new VSTGUI::CBitmap(CResourceDescription(id));
+#endif
```

Comment:
Couldn't you always use CScalableBitmap even if it does not make difference? Do not understand this complexity...

Reply:
The "if mac" has to go somewhere until we sort out resources and loading them on windows and linux. I put it outside the class. I could put it inside the class. Outside the class seemed cleaner.